src/client/Client.cc

```diff
@@ -5964,6 +5966,16 @@ void Client::tick()
     }
   }
 
+  {
+    chrono::duration<double, std::milli> ticks_past_since_req =
+     mono_clock::now() - mono_time_curr_req_stamp;
+    auto int_curr_req_stamp = chrono::duration_cast<std::chrono::seconds>(
+      mono_time_curr_req_stamp - mono_clock::zero());
+    if (mounted && !mds_requests.empty() && ticks_past_since_req.count() >
+	MAX_TIME_FOR_A_REQ)
+      objecter->get_latest_version(0, int_curr_req_stamp.count(), NULL);
```

Comment:
the second argument of get_latest_version() is epoch_t. why stamp here?

Reply:
I think it's better to call Objecter::maybe_request_map() here.

Reply:
+1 `Objecter::maybe_request_map()`

@rishabh-d-dave

Reply:
@rishabh-d-dave @ukernel's suggested change fixes your issue.

The problem was an assertion in the client:

````
     0> 2018-04-17 18:09:34.172 7f79c43cac00 -1 /home/pdonnell/ceph/src/osdc/Objecter.cc: In function 'virtual Objecter::~Objecter()' thread 7f79c43cac00 time 2018-04-17 18:09:34.173531
/home/pdonnell/ceph/src/osdc/Objecter.cc: 5008: FAILED assert(waiting_for_map.empty())

 ceph version 13.0.2-975-ga749882 (a74988218e522d6ecf9b98289539e0c615c14d56) mimic (dev)
 1: (ceph::__ceph_assert_fail(char const*, char const*, int, char const*)+0x102) [0x7f79bb33fb42]
 2: (()+0x292d17) [0x7f79bb33fd17]
 3: (Objecter::~Objecter()+0x158f) [0x7f79bb660b2f]
 4: (Objecter::~Objecter()+0x9) [0x7f79bb660b89]
 5: (StandaloneClient::~StandaloneClient()+0x2e) [0x866055787e]
 6: (StandaloneClient::~StandaloneClient()+0x9) [0x86605578b9]
 7: (main()+0x586) [0x86604dffd6]
 8: (__libc_start_main()+0xf0) [0x7f79ba03a830]
 9: (_start()+0x29) [0x86604e24a9]
 NOTE: a copy of the executable, or `objdump -rdS <executable>` is needed to interpret this.

````

An abort signal (`SIGABRT`) is returned as a `-6` return code by Python's subprocess library.

Reply:
@batrick @ukernel Done.